client/my-sites/plugins/plugin-navbar-header/index.jsx

```diff
@@ -0,0 +1,662 @@
+														</svg>
+													</a>
+												</li>
+											</ul>
+										</div>
+									</li>
+									<li className="x-nav-item x-nav-item--wide">
+										<button
+											role="menuitem"
+											className="x-nav-link x-link"
+											data-dropdown-trigger="features"
+											aria-haspopup="true"
+											aria-expanded="false"
+										>
+											{ translate( 'Features' ) }{ ' ' }
+											<span className="x-nav-link-chevron" aria-hidden="true"></span>
+										</button>
+										<div
+											className="x-dropdown-content"
+											data-dropdown-name="features"
+											role="menu"
+											aria-label={ translate( ' Features' ) }
+											aria-hidden="true"
+										>
+											<ul>
+												<li>
+													<a
+														role="menuitem"
+														className="x-dropdown-link x-link"
+														href="//wordpress.com/features/"
+														title={ translate( ' Features' ) }
+														tabIndex="-1"
+													>
+														{ translate( 'Overview' ) }
+													</a>
+												</li>
+											</ul>
+											<div className="x-dropdown-content-separator"></div>
+											<ul>
+												<li>
+													<a
+														role="menuitem"
+														className="x-dropdown-link x-link"
+														href="//wordpress.com/themes/"
+														title={ translate( 'WordPress Themes' ) }
+														tabIndex="-1"
+													>
+														{ translate( 'WordPress Themes' ) }
+													</a>
+												</li>
+												<li>
+													<a
+														role="menuitem"
+														className="x-dropdown-link x-link"
+														href="//wordpress.com/install-plugins/"
+														title={ translate( 'WordPress Plugins' ) }
+														tabIndex="-1"
+													>
+														{ translate( 'WordPress Plugins' ) }
+													</a>
+												</li>
+												<li>
+													<a
+														role="menuitem"
+														className="x-dropdown-link x-link"
+														href="//wordpress.com/google/"
+														title={ translate( ' Google Apps' ) }
+														tabIndex="-1"
+													>
+														{ translate( 'Google Apps' ) }
+													</a>
+												</li>
+											</ul>
+										</div>
+									</li>
+									<li className="x-nav-item x-nav-item--wide">
+										<button
+											role="menuitem"
+											className="x-nav-link x-link"
+											data-dropdown-trigger="resources"
+											aria-haspopup="true"
+											aria-expanded="false"
+										>
+											{ translate( 'Resources' ) }{ ' ' }
+											<span className="x-nav-link-chevron" aria-hidden="true"></span>
+										</button>
+										<div
+											className="x-dropdown-content"
+											data-dropdown-name="resources"
+											role="menu"
+											aria-label={ translate( ' Resources' ) }
```

Comment:
I'd suggest auditing all new strings for leading and trailing spaces, because those could sometimes be difficult to find and they could introduce subtle unwanted spacing here and there.